net-wireless/sdrpp/sdrpp-1.2.0_p20240117.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR=emake
+inherit cmake
+
+DESCRIPTION="Runtime dependencies for SDRPlusPlus"
+HOMEPAGE="
+	https://www.sdrpp.org/
+	https://github.com/AlexandreRouma/SDRPlusPlus/
+	"
+
+SRC_URI="https://github.com/jharvell/SDRPlusPlus/archive/refs/tags/v${PV}.tar.gz -> ${P}.tgz"
+
+LICENSE="GPL-3"
+SLOT=0
+KEYWORDS="~amd64 ~x86"
+IUSE="+airspy atv +audio-sink +audio-src +bladerf +hackrf +hermes +limesdr +m17 +plutosdr portaudio-sink +portaudio-new-sink +rtlsdr +sched +sdrplay soapysdr uhd"
+PATCHES=(
+	"${FILESDIR}/${PN}-multilib-strict.patch"
+)
+
+REQUIRED_USE="
+	audio-sink? ( !portaudio-sink )
+"
+BDEPEND="dev-build/cmake"
+RDEPEND="
+	sci-libs/fftw:3.0
+	sci-libs/volk
+	app-arch/zstd
+	media-libs/glfw
+	airspy? ( net-wireless/airspy )
+	audio-src? ( media-libs/rtaudio )
+	audio-sink? ( media-libs/rtaudio )
+	bladerf? (
+			net-wireless/bladerf
+			net-wireless/bladerf-firmware
+			net-wireless/bladerf-fpga
+		)
+	hackrf? ( net-libs/libhackrf )
+	limesdr? ( net-wireless/limesuite )
+	m17? ( media-libs/codec2 )
+	plutosdr? (
+			net-libs/libiio
+			net-libs/libad9361-iio
+		)
+	portaudio-sink? ( media-libs/portaudio )
+	portaudio-new-sink? ( media-libs/portaudio )
+	rtlsdr? ( net-wireless/rtl-sdr )
+	sdrplay? ( net-wireless/sdrplay )
+	soapysdr? ( net-wireless/soapysdr )
+	uhd? ( net-wireless/uhd )
+"
+
+src_unpack() {
+	unpack "${A}"
+	mv SDRPlusPlus-"${PV}" "${P}"
```

Comment:
You could instead defining ```src_unpack()``` also just set globally ```S=``` to fit the right name scheme.